flink-runtime/src/main/java/org/apache/flink/runtime/state/UserFacingTemporalListState.java

```diff
@@ -96,4 +97,14 @@ public void clear(long time) {
         originalState.setCurrentNamespace(time);
         originalState.clear();
     }
+
+    @Override
+    public void clearAll() {
+        TemporalNavigator navigator = Preconditions.checkNotNull(internalState.getNavigator());
+        try {
+            navigator.clearAll();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Why don't you bubble up the checked exception?

Reply:
also, seems other RocksDB state backend code is using `FlinkRuntimeException` here...so maybe just that?

Reply:
`RocksDBMapState#clear` on the other hand just prints the exception stacktrace...not sure what is best here